Jellyfin.Plugin.Tvdb/Providers/TvdbSeriesProvider.cs

```diff
@@ -455,6 +455,19 @@ private static void MapSeriesToResult(MetadataResult<Series> result, SeriesExten
             // series.CommunityRating = (float?)tvdbSeries.SiteRating;
             // Attempts to default to USA if not found
             series.OfficialRating = tvdbSeries.ContentRatings.FirstOrDefault(x => string.Equals(x.Country, TvdbCultureInfo.GetCountryInfo(info.MetadataCountryCode)?.ThreeLetterISORegionName, StringComparison.OrdinalIgnoreCase))?.Name ?? tvdbSeries.ContentRatings.FirstOrDefault(x => string.Equals(x.Country, "usa", StringComparison.OrdinalIgnoreCase))?.Name;
+            if (tvdbSeries.Lists is not null && tvdbSeries.Lists is JsonElement jsonElement)
+            {
+                var collections = jsonElement.Deserialize<List<object>>();
+                if (collections is not null)
+                {
+                    var collectionIds = collections.OfType<JsonElement>()
+                        .Where(x => x.GetProperty("isOfficial").GetBoolean())
+                        .Select(x => x.GetProperty("id").GetInt32().ToString(CultureInfo.InvariantCulture))
```

Comment:
This looks super clean but this would throw an exception if a property is missing or not a bool/int

Can't rely on tvdb to be consistent 🙃

Reply:
As far as I can tell, these two property is always present.
I can wrap a try catch around this.